cmd/tetra/tracingpolicy/tracingpolicy.go

```diff
@@ -144,7 +144,33 @@ func New() *cobra.Command {
 					}
 
 					sensors := strings.Join(pol.Sensors, ",")
-					cmd.Printf("%d %s (%s) %s %s\n", pol.Id, pol.Name, pol.Info, namespace, sensors)
+
+					// Previously, enabled, filterID and error were bundled in a
+					// string. To have a retro-compatible tetra command, we scan
+					// the string. If the scan fails, it means something else
+					// might be in Info and we print it.
```

Comment:
Let's make a note of the version that required this (0.11) so that we can remove it eventually.